google/ggrc-core
# GGRC-6622 Disable import of external models

# Issue description

Disable import of external models.

# Steps to test the changes

1. Checkout dev branch
2. Reset DB
3. Create 3 Control
4. Export Controls
5. Checkout this branch
6. Reset DB
7. Import Controls using exported CSV
8. Controls should raise error with message "Trying to import external model entry."

# Solution description

Added checking to ImportRowConverter (src/ggrc/converters/base_row.py:ImportRowConverter._check_object_class).

# Sanity checklist




# PR Review checklist
